target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -46,6 +46,9 @@ ramips_setup_interfaces()
 	ubnt,edgerouter-x-sfp)
 		ucidef_set_interface_lan "eth0 eth1 eth2 eth3 eth4"
 		;;
+	asus,rp-ac56)
+		ucidef_set_interface_lan "lan"
+		;;
```

Comment:
This can be merged with the block above (starting with mikrotik,routerboard-m11g|\). Due to alphabetic sorting, you then have to additionally move the entire block upwards.

Reply:
Done